### PR TITLE
Awesome Books with ES6

### PR DESCRIPTION
**Pull request by degisew**
### Awesome Books with ES6
 **In this project I implemented the following features:**
- I created a new directory and initialized it with git.
- I used the npm init -y command to create the package.json file.
- I created the entry point for my JavaScript code called index.js.
- I created an index.css file for my styles. and main index.html file for project skeleton.
- I created a directory called modules and put all script module files there.
- I used **ES6 syntax.**
- I divided my code into modules and Save each module in a separate file in the modules dir. 
- I Kept all functionalities of the app without errors.
- I Refactored my methods and functions to arrow functions.
- I used **_let_** and **_const_** in the correct way.
- I displayed the current **date** and **time** below the navigation bar and For this requirement, I used a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)

### CODE REVIEW

File: ES6/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import {
+  navBar, listLink, addLink, contactLink,
+} from '../modules/navBar.js';
+// import { Book } from "../modules/addRemoveBook.js";
```
Comment (Nemwel-Boniface) [inline ES6/index.js]: - [ ] Kindly remove all commented out and unused/ unwanted pieces of code from your code base. It makes it look cluttered and messy.

File: ES6/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { anyRandomName, book, bookList } from '../modules/listOfBooks.js';
+import { luxon } from '../modules/luxon.js';
+
+const section = document.querySelector('.container');
+const mainForm = document.getElementById('book-form');
+const currentDate = document.createElement('div');
+currentDate.id = 'displayDateTime';
+
+// Imported from navBar.js
+// navBar;
```
Comment (Nemwel-Boniface) [inline ES6/index.js]: - [ ] Kindly remove all commented out and unused/ unwanted pieces of code from your code base. It makes it look cluttered and messy.

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @degisew               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: ES6/index.js (hunk 2 above)
Comment (degisew) [reply ES6/index.js]: Okay. thank you @Nemwel-Boniface. I will do that.

**Pull request merged**